modules/carto/src/api/maps-v3-client.js

```diff
@@ -168,5 +167,6 @@ export async function getData({type, source, connection, credentials, geoColumn,
 
   const {accessToken} = localCreds;
 
-  return await request({url, format: mapFormat, accessToken});
+  const data = await request({url, format: mapFormat, accessToken});
+  return {data, format: mapFormat};
```

Comment:
You need to modify this example: https://deck.gl/docs/api-reference/carto/overview#support-for-other-deckgl-layers

Reply:
Done: https://github.com/visgl/deck.gl/pull/6202/commits/f76ee9d8b3f3530dd80afe9933f407e7296d0861